helpers/withUserContainer.ts

```diff
@@ -0,0 +1,15 @@
+import { Context } from '../@types/helpers'
+
+export const withUserContainer =
+  <Type, ArgsType>(
+    resolver: (_parent: void, args: ArgsType, ctx: Context) => Type
+  ) =>
+  async (_parent: void, args: ArgsType, ctx: Context) => {
+    const { req } = ctx
+
+    const authorId = req.user?.id
```

Comment:
i believe the team decided to use lodash get instead of optional chaining for this repo